pkg/porter/create.go

```diff
@@ -4,40 +4,59 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"get.porter.sh/porter/pkg"
 	"get.porter.sh/porter/pkg/config"
 )
 
-func (p *Porter) Create() error {
-	fmt.Fprintln(p.Out, "creating porter configuration in the current directory")
+func (p *Porter) Create(bundleName string) error {
+	// Normalize the bundleName by removing trailing slashes
+	bundleName = strings.TrimSuffix(bundleName, "/")
 
-	err := p.CopyTemplate(p.Templates.GetManifest, config.Name)
+	if bundleName == "" {
+		bundleName = p.FileSystem.Getwd() // Use the current directory if no directory is passed
+	}
+
+	fmt.Fprintf(p.Out, "creating porter configuration in %s\n", bundleName)
```

Comment:
I feel like this should be moved to the end of the function, so if we succeed, this outputs

Reply:
Okay, will move it.